webrender/src/tiling.rs

```diff
@@ -1682,20 +1702,21 @@ pub struct StackingContext {
     composite_ops: CompositeOps,
     clip_source: ClipSource,
     clip_cache_info: Option<MaskCacheInfo>,
+    packed_layer_index: PackedLayerIndex,
 }
 
 #[derive(Debug, Clone)]
 #[repr(C)]
-pub struct PackedStackingContext {
+pub struct PackedLayer {
     transform: LayerToWorldTransform,
     inv_transform: WorldToLayerTransform,
     local_clip_rect: LayerRect,
     screen_vertices: [WorldPoint4D; 4],
 }
 
-impl Default for PackedStackingContext {
-    fn default() -> PackedStackingContext {
-        PackedStackingContext {
+impl Default for PackedLayer {
```

Comment:
I believe the semantics of `Default` is misused here. It's not a "default" layer, it's an empty one. `Default` is applicable where you could drop it if you don't care, and it's going to somewhat work. This is not the case here.